### ISSUE TITLE
Crash on 7.4 :[__NSCFString stringByAppendingString:] : [IBGDiskPersistenceManager persistDuplicateOfFileAtPath:]

### ISSUE THREAD

**Issue by alaeddineG** [Instabug/Instabug-iOS#220]
Hi, this a crash we received 2 days ago.

```
*** -[__NSCFString stringByAppendingString:]: nil argument -[IBGDiskPersistenceManager persistDuplicateOfFileAtPath:] (IBGDiskPersistenceManager.m:0)
```

```
Thread 0 [Crashed]:

0    CoreFoundation                           0x184326364     __NSMS6 + 252
1    libobjc.A.dylib                          0x18356c528     objc_exception_throw + 56
2    CoreFoundation                           0x1843262ac     -[NSException initWithCoder:] + 0
3    Foundation                               0x184c21610     -[NSString stringByAppendingString:] + 112
4    InstabugCore                             0x101a0ca80     -[IBGDiskPersistenceManager persistDuplicateOfFileAtPath:] (IBGDiskPersistenceManager.m:0)
5    InstabugCore                             0x101a46760     -[IBGFileLocalAttachment prepareAttachmentPath] (IBGFileLocalAttachment.m:34)
6    InstabugCore                             0x101a468dc     -[IBGFileLocalAttachment prepareAttachmentAndSaveWithCompletion:] (IBGFileLocalAttachment.m:43)
7    Instabug                                 0x10186d42c     -[IBGCrash addAttachment:] (IBGCrash.m:80)
8    Instabug                                 0x10186d2c4     -[IBGCrash initWithTempCrash:] (IBGCrash.m:64)
9    Instabug                                 0x10187e70c     -[IBGCrashManager crashFromTempCrash:] (IBGCrashManager.m:183)
10   Instabug                                 0x10187e518     -[IBGCrashManager handlePendingCrashReportIfExists] (IBGCrashManager.m:0)
11   Instabug                                 0x10187de50     __48-[IBGCrashManager handlePendingCrashesOperation]_block_invoke (IBGCrashManager.m:83)
12   InstabugCore                             0x101a0f8e4     -[IBGOperation start] (IBGOperation.m:97)
13   Foundation                               0x184cff894     __NSOQSchedule_f + 404
14   libdispatch.dylib                        0x183ca2a14     _dispatch_client_callout + 16
15   libdispatch.dylib                        0x183caaf08     _dispatch_continuation_pop$VARIANT$mp + 428
16   libdispatch.dylib                        0x183ca980c     _dispatch_async_redirect_invoke$VARIANT$mp + 604
17   libdispatch.dylib                        0x183cafcf4     _dispatch_root_queue_drain + 600
18   libdispatch.dylib                        0x183cafa38     _dispatch_worker_thread3 + 120
19   libsystem_pthread.dylib                  0x183f4b06c     _pthread_wqthread + 1268
20   libsystem_pthread.dylib                  0x183f4ab6c     start_wqthread + 4
```

### Steps to Reproduce the Problem

Occurred only once . Seems to be happening while attaching something a file when a crash/bug is reported. 
### Expected Behaviour

Doesn't crash.

### Actual Behaviour

Random Crash.

### Instabug integration code

```
- (void) initInstabug{
    if(INSTABUG_ENABLED){
        [Instabug startWithToken:INSTABUG_KEY invocationEvent:IBGInvocationEventShake];
        [Instabug setShakingThresholdForiPhone:3.5 foriPad:3.5];
        [Instabug setColorTheme:IBGColorThemeLight];
        [Instabug setIntroMessageEnabled:NO];
        [Instabug setCommentFieldRequired:YES];
        [Instabug setEmailFieldRequired:YES];
        [Instabug setPrimaryColor:GREEN_COLOR];
        
        [Instabug setValue:@"Report a bug" forStringWithKey:kIBGReportBugStringName];
        [Instabug setValue:@"Questions and Feedback" forStringWithKey:kIBGReportFeedbackStringName];

        NSString* filePath = [self logsPath];
        NSURL* filePathUrl = [NSURL fileURLWithPath:filePath];
        [Instabug addFileAttachmentWithURL:filePathUrl];
    }
}

```

### SDK Version
7.4
### iOS Version
11.2.1
### Device Type
iPhone 7 Plus

**Comment by Kmohamed** [comment]
Hello @alaeddineG this crash is fixed in the latest release v7.6 could you please update to this version.

**Comment by alaeddineG** [comment]
Thank you. Will do.
